Rename toSpan() / bytes() to span(), and vector() free function to makeVector()

cdumez opened the pull request:
#### c614496679dcd71dcd262219b126183fb4de86cd
<pre>
Rename toSpan() / bytes() to span(), and vector() free function to makeVector()
<a href="https://bugs.webkit.org/show_bug.cgi?id=271383">https://bugs.webkit.org/show_bug.cgi?id=271383</a>

Reviewed by Darin Adler.

Rename toSpan() / bytes() to span(), and vector() free function to makeVector(),
for consistency.

* Source/JavaScriptCore/runtime/ArrayBuffer.h:
* Source/JavaScriptCore/runtime/ArrayBufferView.h:
(JSC::ArrayBufferView::span const):
(JSC::ArrayBufferView::vector const):
(JSC::ArrayBufferView::bytes const): Deleted.
* Source/WTF/wtf/FileSystem.h:
(WTF::FileSystemImpl::MappedFileData::span):
(WTF::FileSystemImpl::MappedFileData::toSpan): Deleted.
* Source/WTF/wtf/SHA1.h:
(WTF::SHA1::addBytes):
* Source/WTF/wtf/UUID.h:
(WTF::UUID::span const):
(WTF::UUID::toSpan const): Deleted.
* Source/WTF/wtf/cf/VectorCF.h:
(WTF::span):
(WTF::vector):
(WTF::toSpan): Deleted.
(WTF::toVector): Deleted.
* Source/WTF/wtf/cocoa/FileSystemCocoa.mm:
(WTF::FileSystemImpl::openTemporaryFile):
* Source/WTF/wtf/cocoa/SpanCocoa.h:
(WTF::span):
(WTF::toSpan): Deleted.
* Source/WTF/wtf/cocoa/VectorCocoa.h:
(WTF::vector):
(WTF::toVector): Deleted.
* Source/WTF/wtf/persistence/PersistentEncoder.h:
(WTF::Persistence::Encoder::span const):
(WTF::Persistence::Encoder::bytes const): Deleted.
* Source/WTF/wtf/text/Base64.h:
(WTF::base64EncodeToVector):
(WTF::base64EncodeToString):
(WTF::base64EncodeToStringReturnNullIfOverflow):
(WTF::base64Encoded):
* Source/WTF/wtf/text/CString.h:
* Source/WTF/wtf/text/WTFString.cpp:
(asciiDebug):
* Source/WebCore/Modules/async-clipboard/ClipboardItem.cpp:
(WebCore::ClipboardItem::blobFromString):
* Source/WebCore/Modules/async-clipboard/ios/ClipboardImageReaderIOS.mm:
(WebCore::ClipboardImageReader::readBuffer):
* Source/WebCore/Modules/async-clipboard/mac/ClipboardImageReaderMac.mm:
(WebCore::ClipboardImageReader::readBuffer):
* Source/WebCore/Modules/encryptedmedia/legacy/LegacyCDMSessionClearKey.cpp:
(WebCore::CDMSessionClearKey::generateKeyRequest):
* Source/WebCore/Modules/fetch/FetchBody.cpp:
(WebCore::FetchBody::consumeArrayBuffer):
(WebCore::FetchBody::consumeArrayBufferView):
(WebCore::FetchBody::bodyAsFormData const):
* Source/WebCore/Modules/fetch/FetchBodyConsumer.cpp:
(WebCore::FetchBodyConsumer::resolveWithFormData):
(WebCore::FetchBodyConsumer::resolve):
(WebCore::FetchBodyConsumer::takeAsText):
* Source/WebCore/Modules/fetch/FetchResponse.cpp:
(WebCore::FetchResponse::Loader::didReceiveData):
(WebCore::FetchResponse::Loader::consumeDataByChunk):
* Source/WebCore/Modules/indexeddb/server/SQLiteIDBBackingStore.cpp:
(WebCore::IDBServer::SQLiteIDBBackingStore::addExistingIndex):
(WebCore::IDBServer::SQLiteIDBBackingStore::createObjectStore):
(WebCore::IDBServer::SQLiteIDBBackingStore::createIndex):
(WebCore::IDBServer::SQLiteIDBBackingStore::uncheckedHasIndexRecord):
(WebCore::IDBServer::SQLiteIDBBackingStore::uncheckedPutIndexRecord):
(WebCore::IDBServer::SQLiteIDBBackingStore::keyExistsInObjectStore):
(WebCore::IDBServer::SQLiteIDBBackingStore::deleteRecord):
(WebCore::IDBServer::SQLiteIDBBackingStore::addRecord):
(WebCore::IDBServer::SQLiteIDBBackingStore::getRecord):
(WebCore::IDBServer::SQLiteIDBBackingStore::getAllObjectStoreRecords):
(WebCore::IDBServer::SQLiteIDBBackingStore::uncheckedGetIndexRecordForOneKey):
(WebCore::IDBServer::SQLiteIDBBackingStore::getCount):
* Source/WebCore/Modules/indexeddb/server/SQLiteIDBCursor.cpp:
(WebCore::IDBServer::SQLiteIDBCursor::bindArguments):
(WebCore::IDBServer::SQLiteIDBCursor::resetAndRebindPreIndexStatementIfNecessary):
* Source/WebCore/Modules/notifications/NotificationDataCocoa.mm:
(WebCore::NotificationData::fromDictionary):
* Source/WebCore/Modules/push-api/PushDatabase.cpp:
(WebCore::uuidToSpan):
* Source/WebCore/Modules/push-api/PushEvent.cpp:
(WebCore::dataFromPushMessageDataInit):
* Source/WebCore/Modules/push-api/PushManager.cpp:
(WebCore::PushManager::subscribe):
* Source/WebCore/Modules/webauthn/AuthenticationExtensionsClientOutputs.cpp:
(WebCore::AuthenticationExtensionsClientOutputs::toCBOR const):
* Source/WebCore/Modules/webauthn/AuthenticatorAttestationResponse.cpp:
(WebCore::coseKeyForAttestationObject):
(WebCore::AuthenticatorAttestationResponse::getAuthenticatorData const):
* Source/WebCore/Modules/webauthn/cbor/CBORValue.cpp:
(cbor::CBORValue::CBORValue):
* Source/WebCore/Modules/webauthn/cbor/CBORWriter.cpp:
(cbor::CBORWriter::encodeCBOR):
* Source/WebCore/Modules/webauthn/fido/Pin.cpp:
(fido::pin::SetPinRequest::tryCreate):
* Source/WebCore/Modules/webauthn/fido/U2fCommandConstructor.cpp:
(fido::WebCore::constructU2fSignCommand):
* Source/WebCore/Modules/webauthn/fido/U2fResponseConverter.cpp:
(fido::readU2fSignResponse):
* Source/WebCore/Modules/webcodecs/WebCodecsAudioData.cpp:
(WebCore::WebCodecsAudioData::create):
* Source/WebCore/Modules/webcodecs/WebCodecsEncodedAudioChunk.cpp:
(WebCore::WebCodecsEncodedAudioChunk::WebCodecsEncodedAudioChunk):
* Source/WebCore/Modules/webcodecs/WebCodecsEncodedVideoChunk.cpp:
(WebCore::WebCodecsEncodedVideoChunk::WebCodecsEncodedVideoChunk):
* Source/WebCore/Modules/websockets/WebSocket.cpp:
(WebCore::WebSocket::didReceiveMessage):
* Source/WebCore/Modules/websockets/WebSocketDeflateFramer.cpp:
(WebCore::WebSocketDeflateFramer::deflate):
(WebCore::WebSocketDeflateFramer::inflate):
* Source/WebCore/Modules/websockets/WebSocketDeflater.h:
(WebCore::WebSocketDeflater::span const):
(WebCore::WebSocketInflater::span const):
(WebCore::WebSocketDeflater::bytes const): Deleted.
(WebCore::WebSocketInflater::bytes const): Deleted.
* Source/WebCore/accessibility/mac/AccessibilityObjectMac.mm:
(WebCore::AXRemoteFrame::generateRemoteToken const):
* Source/WebCore/bindings/js/BufferSource.h:
(WebCore::BufferSource::span const):
(WebCore::BufferSource::bytes const): Deleted.
* Source/WebCore/bindings/js/JSDOMGlobalObject.cpp:
(WebCore::handleResponseOnStreamingAction):
* Source/WebCore/bindings/js/ScriptBufferSourceProvider.h:
* Source/WebCore/bindings/js/SerializedScriptValue.cpp:
(WebCore::CloneSerializer::dumpImageBitmap):
(WebCore::CloneSerializer::dumpIfTerminal):
(WebCore::CloneSerializer::write):
* Source/WebCore/contentextensions/ContentExtensionActions.cpp:
(WebCore::ContentExtensions::append):
* Source/WebCore/contentextensions/ContentExtensionStringSerialization.cpp:
(WebCore::ContentExtensions::serializeString):
* Source/WebCore/crypto/SubtleCrypto.cpp:
(WebCore::toKeyData):
(WebCore::copyToVector):
(WebCore::SubtleCrypto::wrapKey):
* Source/WebCore/crypto/cocoa/SerializedCryptoKeyWrapMac.mm:
(WebCore::wrapSerializedCryptoKey):
(WebCore::unwrapSerializedCryptoKey):
* Source/WebCore/crypto/parameters/CryptoAlgorithmAesCbcCfbParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmAesCtrParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmAesGcmParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmHkdfParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmPbkdf2Params.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmRsaKeyGenParams.h:
(WebCore::CryptoAlgorithmRsaKeyGenParams::publicExponentVector const):
* Source/WebCore/crypto/parameters/CryptoAlgorithmRsaOaepParams.h:
* Source/WebCore/dom/TextDecoder.cpp:
(WebCore::TextDecoder::decode):
* Source/WebCore/editing/cocoa/WebContentReaderCocoa.mm:
(WebCore::sanitizeMarkupWithArchive):
* Source/WebCore/fileapi/BlobBuilder.cpp:
(WebCore::BlobBuilder::append):
* Source/WebCore/inspector/agents/InspectorPageAgent.cpp:
(WebCore::InspectorPageAgent::mainResourceContent):
(WebCore::InspectorPageAgent::sharedBufferContent):
* Source/WebCore/loader/appcache/ApplicationCacheStorage.cpp:
(WebCore::ApplicationCacheStorage::store):
* Source/WebCore/loader/archive/mhtml/MHTMLParser.cpp:
(WebCore::MHTMLParser::parseNextPart):
* Source/WebCore/loader/cache/CachedFont.cpp:
(WebCore::CachedFont::policyForCustomFont):
* Source/WebCore/loader/cache/CachedScript.cpp:
(WebCore::CachedScript::script):
* Source/WebCore/platform/SharedBuffer.cpp:
(WebCore::combineSegmentsData):
(WebCore::FragmentedSharedBuffer::getContiguousData const):
(WebCore::FragmentedSharedBuffer::read const):
* Source/WebCore/platform/SharedBuffer.h:
(WebCore::DataSegment::span const):
(WebCore::SharedBuffer::span const):
(WebCore::DataSegment::bytes const): Deleted.
(WebCore::SharedBuffer::bytes const): Deleted.
* Source/WebCore/platform/cocoa/ContentFilterUnblockHandlerCocoa.mm:
(WebCore::ContentFilterUnblockHandler::webFilterEvaluatorData const):
* Source/WebCore/platform/graphics/ImageBackingStore.h:
(WebCore::ImageBackingStore::ImageBackingStore):
* Source/WebCore/platform/graphics/WOFFFileFormat.cpp:
(WebCore::convertWOFFToSfnt):
* Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateAVFoundationObjC.mm:
(WebCore::MediaPlayerPrivateAVFoundationObjC::shouldWaitForLoadingOfResource):
* Source/WebCore/platform/graphics/cocoa/WebMAudioUtilitiesCocoa.mm:
(WebCore::cookieFromOpusCookieContents):
* Source/WebCore/platform/graphics/coretext/FontCustomPlatformDataCoreText.cpp:
(WebCore::FontCustomPlatformData::serializedData const):
* Source/WebCore/platform/graphics/coretext/FontPlatformDataCoreText.cpp:
(WebCore::FontPlatformData::platformSerializationData const):
* Source/WebCore/platform/mediastream/mac/AVVideoCaptureSource.mm:
(WebCore::AVVideoCaptureSource::captureOutputDidFinishProcessingPhoto):
* Source/WebCore/platform/network/FormData.cpp:
(WebCore::FormData::create):
(WebCore::appendBlobResolved):
* Source/WebCore/platform/network/FormDataBuilder.cpp:
(WebCore::FormDataBuilder::append):
* Source/WebCore/platform/network/SynchronousLoaderClient.cpp:
(WebCore::SynchronousLoaderClient::didReceiveData):
* Source/WebCore/platform/network/cocoa/ResourceRequestCocoa.mm:
(WebCore::ResourceRequest::doUpdateResourceHTTPBody):
* Source/WebCore/platform/win/ClipboardUtilitiesWin.cpp:
(WebCore::append):
* Source/WebCore/testing/Internals.cpp:
(WebCore::Internals::deserializeBuffer const):
(WebCore::Internals::createPushSubscription):
* Source/WebCore/testing/MockContentFilter.cpp:
(WebCore::MockContentFilter::maybeDetermineStatus):
* Source/WebCore/testing/ServiceWorkerInternals.cpp:
(WebCore::ServiceWorkerInternals::schedulePushEvent):
(WebCore::ServiceWorkerInternals::createPushSubscription):
* Source/WebCore/workers/service/background-fetch/BackgroundFetch.cpp:
(WebCore::BackgroundFetch::doStore):
* Source/WebCore/xml/XMLHttpRequest.cpp:
(WebCore::XMLHttpRequest::send):
* Source/WebCore/xml/parser/XMLDocumentParserLibxml2.cpp:
(WebCore::openFunc):
* Source/WebGPU/WGSL/ConstantFunctions.h:
(WGSL::constantMatrix):
(WGSL::CONSTANT_FUNCTION):
* Source/WebGPU/WGSL/ConstantValue.h:
(WGSL::ConstantValue::vector const):
(WGSL::ConstantValue::toVector const): Deleted.
* Source/WebGPU/WGSL/Types.cpp:
(WGSL::conversionRank):
* Source/WebKit/NetworkProcess/Downloads/cocoa/DownloadCocoa.mm:
(WebKit::Download::platformCancelNetworkLoad):
* Source/WebKit/NetworkProcess/NetworkResourceLoader.cpp:
(WebKit::sendReplyToSynchronousRequest):
* Source/WebKit/NetworkProcess/cocoa/NetworkSessionCocoa.mm:
(-[WKNetworkSessionDelegate URLSession:task:didCompleteWithError:]):
(WebKit::NetworkSessionCocoa::setProxyConfigData):
* Source/WebKit/NetworkProcess/cocoa/WebSocketTaskCocoa.mm:
(WebKit::WebSocketTask::readNextMessage):
* Source/WebKit/NetworkProcess/curl/WebSocketTaskCurl.cpp:
(WebKit::WebSocketTask::appendReceivedBuffer):
(WebKit::WebSocketTask::sendClosingHandshakeIfNeeded):
* Source/WebKit/NetworkProcess/storage/CacheStorageDiskStore.cpp:
(WebKit::CacheStorageDiskStore::readAllRecordInfos):
(WebKit::CacheStorageDiskStore::readRecords):
(WebKit::encodeRecordHeader):
(WebKit::encodeRecordBody):
(WebKit::encodeRecord):
* Source/WebKit/NetworkProcess/storage/CacheStorageManager.cpp:
(WebKit::writeSizeFile):
* Source/WebKit/Shared/API/APIData.h:
(API::Data::span const):
(API::Data::bytes const): Deleted.
* Source/WebKit/Shared/API/APIData.serialization.in:
* Source/WebKit/Shared/API/Cocoa/WKRemoteObjectCoder.mm:
(-[WKRemoteObjectDecoder decodeBytesForKey:returnedLength:]):
* Source/WebKit/Shared/API/c/WKData.cpp:
(WKDataGetBytes):
* Source/WebKit/Shared/APIWebArchive.mm:
(API::WebArchive::WebArchive):
* Source/WebKit/Shared/APIWebArchiveResource.mm:
(API::WebArchiveResource::WebArchiveResource):
* Source/WebKit/Shared/Cocoa/CoreIPCCFURL.h:
* Source/WebKit/Shared/Cocoa/CoreIPCCFURL.mm:
(WebKit::CoreIPCCFURL::vector const):
(WebKit::CoreIPCCFURL::bytes const): Deleted.
* Source/WebKit/Shared/Cocoa/CoreIPCCFURL.serialization.in:
* Source/WebKit/Shared/Cocoa/SandboxExtensionCocoa.mm:
(WebKit::SandboxExtension::createHandleForTemporaryFile):
* Source/WebKit/Shared/Cocoa/WKNSData.mm:
(-[WKNSData bytes]):
* Source/WebKit/Shared/Cocoa/WebPushMessageCocoa.mm:
(WebKit::WebPushMessage::fromDictionary):
* Source/WebKit/Shared/WTFArgumentCoders.serialization.in:
* Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in:
* Source/WebKit/Shared/WebPushMessage.cpp:
(WebKit::WebPushMessage::notificationPayloadToCoreData const):
* Source/WebKit/Shared/cf/CookieStorageUtilsCF.mm:
(WebKit::identifyingDataFromCookieStorage):
* Source/WebKit/Shared/mac/AuxiliaryProcessMac.mm:
(WebKit::compileAndCacheSandboxProfile):
* Source/WebKit/UIProcess/API/C/WKNotification.cpp:
(WKNotificationCopyCoreIDForTesting):
* Source/WebKit/UIProcess/API/C/WKNotificationManager.cpp:
(WKNotificationManagerProviderDidClickNotification_b):
* Source/WebKit/UIProcess/API/C/WKPage.cpp:
(WKPageLoadData):
(WKPageLoadDataWithUserData):
(restoreFromSessionState):
* Source/WebKit/UIProcess/API/C/WKSessionStateRef.cpp:
(WKSessionStateCreateFromData):
* Source/WebKit/UIProcess/API/Cocoa/WKProcessPool.mm:
(-[WKProcessPool _setObject:forBundleParameter:]):
(-[WKProcessPool _setObjectsForBundleParametersWithDictionary:]):
* Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm:
(-[WKWebView _restoreFromSessionStateData:]):
* Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataStore.mm:
(-[WKWebsiteDataStore setProxyConfigurations:]):
* Source/WebKit/UIProcess/API/Cocoa/_WKWebAuthenticationPanel.mm:
(getAllLocalAuthenticatorCredentialsImpl):
(+[_WKWebAuthenticationPanel setDisplayNameForLocalCredentialWithGroupAndID:credential:displayName:]):
(+[_WKWebAuthenticationPanel setNameForLocalCredentialWithGroupAndID:credential:name:]):
(+[_WKWebAuthenticationPanel importLocalAuthenticatorWithAccessGroup:credential:error:]):
(+[_WKWebAuthenticationPanel exportLocalAuthenticatorCredentialWithGroupAndID:credential:error:]):
(-[_WKWebAuthenticationPanel makeCredentialWithMediationRequirement:clientDataHash:options:completionHandler:]):
(-[_WKWebAuthenticationPanel getAssertionWithMediationRequirement:clientDataHash:options:completionHandler:]):
(+[_WKWebAuthenticationPanel encodeMakeCredentialCommandWithClientDataHash:options:userVerificationAvailability:authenticatorSupportedExtensions:]):
(+[_WKWebAuthenticationPanel encodeGetAssertionCommandWithClientDataHash:options:userVerificationAvailability:authenticatorSupportedExtensions:]):
* Source/WebKit/UIProcess/API/glib/WebKitFaviconDatabase.cpp:
(webkitFaviconDatabaseSetIconForPageURL):
* Source/WebKit/UIProcess/API/glib/WebKitWebResource.cpp:
(resourceDataCallback):
(webkit_web_resource_get_data_finish):
* Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp:
(getContentsAsMHTMLDataCallback):
(webkit_web_view_save_finish):
* Source/WebKit/UIProcess/Cocoa/LegacyCustomProtocolManagerClient.mm:
(-[WKCustomProtocolLoader connection:didReceiveData:]):
* Source/WebKit/UIProcess/Cocoa/SOAuthorization/SubFrameSOAuthorizationSession.mm:
(WebKit::SubFrameSOAuthorizationSession::completeInternal):
* Source/WebKit/UIProcess/Notifications/WebNotificationManagerProxy.cpp:
(WebKit::WebNotificationManagerProxy::providerDidCloseNotifications):
* Source/WebKit/UIProcess/WebAuthentication/Cocoa/CcidConnection.mm:
(WebKit::CcidConnection::transact const):
* Source/WebKit/UIProcess/WebAuthentication/Cocoa/LocalAuthenticator.mm:
(WebKit::LocalAuthenticatorInternal::getExistingCredentials):
(WebKit::LocalAuthenticator::processLargeBlobExtension):
(WebKit::LocalAuthenticator::continueMakeCredentialAfterAttested):
* Source/WebKit/UIProcess/WebAuthentication/Cocoa/NfcConnection.mm:
(WebKit::NfcConnection::transact const):
* Source/WebKit/UIProcess/WebAuthentication/Cocoa/WebAuthenticatorCoordinatorProxy.mm:
(WebKit::toExtensionOutputs):
* Source/WebKit/UIProcess/WebAuthentication/Virtual/VirtualAuthenticatorUtils.mm:
(WebKit::signatureForPrivateKey):
* Source/WebKit/UIProcess/WebPageProxy.cpp:
(WebKit::WebPageProxy::didChooseFilesForOpenPanelWithDisplayStringAndIcon):
(WebKit::WebPageProxy::getWebCryptoMasterKey):
* Source/WebKit/UIProcess/WebURLSchemeTask.cpp:
(WebKit::WebURLSchemeTask::didComplete):
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp:
(WebKit::WebsiteDataStore::resumeDownload):
* Source/WebKit/UIProcess/ios/WKContentView.mm:
(-[WKContentView _accessibilityRegisterUIProcessTokens]):
* Source/WebKit/UIProcess/mac/WKPrintingView.mm:
(-[WKPrintingView _preparePDFDataForPrintingOnSecondaryThread]):
* Source/WebKit/UIProcess/mac/WKSharingServicePickerDelegate.mm:
(-[WKSharingServicePickerDelegate sharingService:didShareItems:]):
* Source/WebKit/UIProcess/mac/WebContextMenuProxyMac.mm:
(WebKit::WebContextMenuProxyMac::removeBackgroundFromControlledImage):
* Source/WebKit/UIProcess/mac/WebViewImpl.mm:
(WebKit::WebViewImpl::accessibilityRegisterUIProcessTokens):
* Source/WebKit/WebProcess/InjectedBundle/API/c/WKBundle.cpp:
(WKBundleCopyWebNotificationID):
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageEditorClient.cpp:
(WebKit::InjectedBundlePageEditorClient::getPasteboardDataForRange):
* Source/WebKit/WebProcess/InjectedBundle/mac/InjectedBundleMac.mm:
(WebKit::createUnarchiver):
* Source/WebKit/WebProcess/Network/WebSocketChannel.cpp:
(WebKit::WebSocketChannel::createMessageQueue):
* Source/WebKit/WebProcess/Network/webrtc/RTCDataChannelRemoteManager.cpp:
(WebKit::RTCDataChannelRemoteManager::RemoteSourceConnection::didReceiveStringData):
* Source/WebKit/WebProcess/Plugins/PDF/PDFIncrementalLoader.mm:
(WebKit::PDFPluginStreamLoaderClient::didReceiveData):
* Source/WebKit/WebProcess/WebPage/Cocoa/WebPageCocoa.mm:
(WebKit::WebPage::accessibilityTransferRemoteToken):
(WebKit::WebPage::bindRemoteAccessibilityFrames):
* Source/WebKit/WebProcess/WebPage/WebFrame.cpp:
(WebKit::WebFrame::source const):
* Source/WebKit/WebProcess/WebPage/mac/WebPageMac.mm:
(WebKit::WebPage::handleSelectionServiceClick):
* Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm:
(WebKit::registerLogHook):
* Source/WebKit/webpushd/ApplePushServiceConnection.mm:
(-[_WKAPSConnectionDelegate connection:didReceivePublicToken:]):
* Source/WebKit/webpushd/WebPushDaemon.mm:
(WebPushD::WebPushDaemon::getPendingPushMessages):
(WebPushD::WebPushDaemon::setPublicTokenForTesting):
* Source/WebKitLegacy/WebCoreSupport/SocketStreamHandle.cpp:
(WebCore::SocketStreamHandle::sendHandshake):
* Source/WebKitLegacy/WebCoreSupport/SocketStreamHandleImpl.cpp:
(WebCore::cookieDataForHandshake):
* Source/WebKitLegacy/WebCoreSupport/WebSocketChannel.cpp:
(WebCore::WebSocketChannel::send):
(WebCore::WebSocketChannel::startClosingHandshake):
(WebCore::WebSocketChannel::processOutgoingFrameQueue):
* Source/WebKitLegacy/mac/WebView/WebFrame.mm:
(+[WebFrame stringWithData:textEncodingName:]):
* Source/WebKitLegacy/mac/WebView/WebHTMLRepresentation.mm:
(-[WebHTMLRepresentation documentSource]):
* Source/WebKitLegacy/mac/WebView/WebResource.mm:
(-[WebResource _stringValue]):
* Tools/TestWebKitAPI/Tests/WTF/cf/VectorCF.cpp:
(TestWebKitAPI::TEST):
* Tools/TestWebKitAPI/Tests/WebCore/CtapResponseTest.cpp:
(TestWebKitAPI::TEST):
* Tools/TestWebKitAPI/Tests/WebCore/KeyedCoding.cpp:
(TestWebKitAPI::TEST):
(TestWebKitAPI::testSimpleValue):
* Tools/TestWebKitAPI/Tests/WebCore/curl/CurlMultipartHandleTests.cpp:
(TestWebKitAPI::Curl::span):
(TestWebKitAPI::Curl::TEST):
(TestWebKitAPI::Curl::toSpan): Deleted.
* Tools/TestWebKitAPI/Tests/WebKitCocoa/EventAttribution.mm:
(TestWebKitAPI::signUnlinkableTokenAndSendSecretToken):
* Tools/TestWebKitAPI/Tests/WebKitCocoa/MediaLoading.mm:
(TestWebKitAPI::testVideoBytes):
(TestWebKitAPI::TEST):
* Tools/TestWebKitAPI/Tests/WebKitCocoa/WebsiteDataStoreCustomPaths.mm:
(respondToRangeRequests):
* Tools/TestWebKitAPI/cocoa/HTTPServer.mm:
(TestWebKitAPI::appendUTF8ToVector):
* Tools/WebKitTestRunner/DataFunctions.h:
(WTR::uuidToData):

Canonical link: <a href="https://commits.webkit.org/276520@main">https://commits.webkit.org/276520@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/d3e6c9b295e9339b359fed6d8840e93768ee5a8b

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/44887 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/23986 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/47377 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/47541 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/40892 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/47190 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/28042 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/21383 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/36856 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/45465 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/21034 "Passed tests") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/38647 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/17943 "Passed tests") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/18446 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/39791 "Passed tests") | [✅ 🛠 wpe-skia](https://ews-build.webkit.org/#/builders/52/builds/2934 "Built successfully") | 
| [  ~~🛠 🧪 jsc~~](https://ews-build.webkit.org/#/builders/20/builds/38087 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/41114 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/40084 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/49212 "Built successfully") | 
| [  ~~🛠 🧪 jsc-arm64~~](https://ews-build.webkit.org/#/builders/12/builds/44343 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/19855 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/50/builds/16396 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/43847 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/46/builds/21174 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/42609 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/43/builds/21512 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/35/builds/51515 "Built successfully") | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/6226 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/20848 "Built successfully") | | [✅ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/25/builds/10454 "Passed tests") | 
<!--EWS-Status-Bubble-End-->

darinadler commented on Source/WTF/wtf/cf/VectorCF.h:
```diff
@@ -188,7 +188,7 @@ inline std::optional<float> makeVectorElement(const float*, CFNumberRef cfNumber
-using WTF::toSpan;
-using WTF::toVector;
+using WTF::span;
+using WTF::vector;
```
One difference is that the `vector` function allocates memory and creates a new object, whereas `span` just offers a view into an existing object without allocating any memory. That’s why I named the other function `makeVector`. Could name it `copyToVector` or even try to overload `makeVector`. But keeping the name just `vector` also seems OK. I also noticed that some of our `vector` functions return references to vectors so those do not allocate memory. I think they might merit different naming?

cdumez replied:
OK, so maybe
For Span: `span()` for both free functions and member functions
For Vector:
- `copyToVector()` both free functions and member functions when creating a new vector (allocating memory)
- `vector()` for member functions when not allocating memory (returning a reference to an internal Vector)

How does this sound?

darinadler replied:
Sounds perfect.

darinadler replied:
I also like your choice of `makeVector`, though. One thing I am not so fond of is `toVector` member functions that allocate a new vector.

cdumez replied:
Yes, would `makeVector()` be better for the member functions then? 

I tried going with my earlier proposal too but it led to some build errors that I didn't have time to figure out (likely because of the existing `copyToVector()` in Vector.h.

cdumez replied:
Or maybe I replace the `vector()` member functions with `makeVector()` free function overloads.
But then why not do the same with `span()`?

darinadler commented on Source/WTF/wtf/cocoa/VectorCocoa.h:
```diff
@@ -113,13 +113,13 @@ template<typename MapFunctionType> Vector<typename std::invoke_result_t<MapFunct
+inline Vector<uint8_t> vector(NSData *data)
 {
-    return toSpan(data);
+    return span(data);
```
In some call sites I see us doing `return { span() }` but here we just `return span()` without the braces.

darinadler commented on Source/WTF/wtf/text/CString.h:
```diff
@@ -78,14 +78,14 @@ class CString final {
     }
 
-    std::span<const uint8_t> bytesInludingNullTerminator() const
+    std::span<const uint8_t> spanInludingNullTerminator() const
```
I just noticed that "including" is misspelled here as "inluding".

darinadler commented on Source/WebCore/crypto/cocoa/SerializedCryptoKeyWrapMac.mm:
```diff
@@ -276,17 +276,17 @@ bool unwrapSerializedCryptoKey(const Vector<uint8_t>& masterKey, const Vector<ui
     if (![tagObject isKindOfClass:[NSData class]])
         return false;
-    Vector<uint8_t> tag = toVector(tagObject);
+    Vector<uint8_t> tag = vector(tagObject);
```
This could be a `span` instead. The copying is unnecessary.

darinadler commented on Source/WebCore/crypto/cocoa/SerializedCryptoKeyWrapMac.mm:
```diff
@@ -276,17 +276,17 @@ bool unwrapSerializedCryptoKey(const Vector<uint8_t>& masterKey, const Vector<ui
     if (![wrappedKEKObject isKindOfClass:[NSData class]])
         return false;
-    Vector<uint8_t> wrappedKEK = toVector(wrappedKEKObject);
+    Vector<uint8_t> wrappedKEK = vector(wrappedKEKObject);
```
This could be a `span` instead. The copying is unnecessary.

darinadler commented on Source/WebCore/crypto/cocoa/SerializedCryptoKeyWrapMac.mm:
```diff
@@ -276,17 +276,17 @@ bool unwrapSerializedCryptoKey(const Vector<uint8_t>& masterKey, const Vector<ui
     if (![encryptedKeyObject isKindOfClass:[NSData class]])
         return false;
-    Vector<uint8_t> encryptedKey = toVector(encryptedKeyObject);
+    Vector<uint8_t> encryptedKey = vector(encryptedKeyObject);
```
This could be a `span` instead. The copying is unnecessary.

darinadler commented on Source/WebCore/platform/cocoa/ContentFilterUnblockHandlerCocoa.mm:
```diff
@@ -87,7 +87,7 @@
     NSError *error { nil };
     NSData *data = [NSKeyedArchiver archivedDataWithRootObject:m_webFilterEvaluator.get() requiringSecureCoding:YES error:&error];
-    return toVector(data);
+    return vector(data);
```
Seems like this could be a a one-liner.

darinadler commented on Source/WebCore/platform/mediastream/mac/AVVideoCaptureSource.mm:
```diff
@@ -1152,7 +1152,7 @@ static inline IntDegrees sensorOrientation(AVCaptureVideoOrientation videoOrient
 
     NSData* data = [photo fileDataRepresentation];
-    resolvePendingPhotoRequest(toVector(data), "image/jpeg"_s);
+    resolvePendingPhotoRequest(vector(data), "image/jpeg"_s);
```
I wonder if the allocation of a `Vector` here is necessary.

darinadler commented on Source/WebCore/testing/Internals.cpp:
```diff
@@ -4214,7 +4214,7 @@ Ref<ArrayBuffer> Internals::serializeObject(const RefPtr<SerializedScriptValue>&
 Ref<SerializedScriptValue> Internals::deserializeBuffer(ArrayBuffer& buffer) const
 {
-    return SerializedScriptValue::createFromWireBytes(buffer.toVector());
+    return SerializedScriptValue::createFromWireBytes(buffer.vector());
```
Seems like the `SerializedScriptValue::createFromWireBytes` function could take a `std::span` instead of a `Vector`.

darinadler commented on Source/WebGPU/WGSL/Types.cpp:
```diff
@@ -229,12 +229,12 @@ ConversionRank conversionRank(const Type* from, const Type* to)
     if (auto* fromVector = std::get_if<Vector>(from)) {
-        auto* toVector = std::get_if<Vector>(to);
-        if (!toVector)
+        auto* vector = std::get_if<Vector>(to);
```
Not sure this one should have been renamed. I suggest tossing the changes to this file.

cdumez replied:
That was a auto replace mistake, will revert changes to this file

darinadler commented on Source/WebKit/Shared/cf/CookieStorageUtilsCF.mm:
```diff
@@ -51,7 +51,7 @@
 
     auto cfData = adoptCF(CFHTTPCookieStorageCreateIdentifyingData(kCFAllocatorDefault, cookieStorage));
-    return toVector(cfData.get());
+    return vector(cfData.get());
```
Seems like this could be a one-liner.

cdumez committed:
Rename toSpan() / bytes() to span(), and vector() free function to makeVector()
https://bugs.webkit.org/show_bug.cgi?id=271383

Reviewed by Darin Adler.

Rename toSpan() / bytes() to span(), and vector() free function to makeVector(),
for consistency.

* Source/JavaScriptCore/runtime/ArrayBuffer.h:
* Source/JavaScriptCore/runtime/ArrayBufferView.h:
(JSC::ArrayBufferView::span const):
(JSC::ArrayBufferView::vector const):
(JSC::ArrayBufferView::bytes const): Deleted.
* Source/WTF/wtf/FileSystem.h:
(WTF::FileSystemImpl::MappedFileData::span):
(WTF::FileSystemImpl::MappedFileData::toSpan): Deleted.
* Source/WTF/wtf/SHA1.h:
(WTF::SHA1::addBytes):
* Source/WTF/wtf/UUID.h:
(WTF::UUID::span const):
(WTF::UUID::toSpan const): Deleted.
* Source/WTF/wtf/cf/VectorCF.h:
(WTF::span):
(WTF::vector):
(WTF::toSpan): Deleted.
(WTF::toVector): Deleted.
* Source/WTF/wtf/cocoa/FileSystemCocoa.mm:
(WTF::FileSystemImpl::openTemporaryFile):
* Source/WTF/wtf/cocoa/SpanCocoa.h:
(WTF::span):
(WTF::toSpan): Deleted.
* Source/WTF/wtf/cocoa/VectorCocoa.h:
(WTF::vector):
(WTF::toVector): Deleted.
* Source/WTF/wtf/persistence/PersistentEncoder.h:
(WTF::Persistence::Encoder::span const):
(WTF::Persistence::Encoder::bytes const): Deleted.
* Source/WTF/wtf/text/Base64.h:
(WTF::base64EncodeToVector):
(WTF::base64EncodeToString):
(WTF::base64EncodeToStringReturnNullIfOverflow):
(WTF::base64Encoded):
* Source/WTF/wtf/text/CString.h:
* Source/WTF/wtf/text/WTFString.cpp:
(asciiDebug):
* Source/WebCore/Modules/async-clipboard/ClipboardItem.cpp:
(WebCore::ClipboardItem::blobFromString):
* Source/WebCore/Modules/async-clipboard/ios/ClipboardImageReaderIOS.mm:
(WebCore::ClipboardImageReader::readBuffer):
* Source/WebCore/Modules/async-clipboard/mac/ClipboardImageReaderMac.mm:
(WebCore::ClipboardImageReader::readBuffer):
* Source/WebCore/Modules/encryptedmedia/legacy/LegacyCDMSessionClearKey.cpp:
(WebCore::CDMSessionClearKey::generateKeyRequest):
* Source/WebCore/Modules/fetch/FetchBody.cpp:
(WebCore::FetchBody::consumeArrayBuffer):
(WebCore::FetchBody::consumeArrayBufferView):
(WebCore::FetchBody::bodyAsFormData const):
* Source/WebCore/Modules/fetch/FetchBodyConsumer.cpp:
(WebCore::FetchBodyConsumer::resolveWithFormData):
(WebCore::FetchBodyConsumer::resolve):
(WebCore::FetchBodyConsumer::takeAsText):
* Source/WebCore/Modules/fetch/FetchResponse.cpp:
(WebCore::FetchResponse::Loader::didReceiveData):
(WebCore::FetchResponse::Loader::consumeDataByChunk):
* Source/WebCore/Modules/indexeddb/server/SQLiteIDBBackingStore.cpp:
(WebCore::IDBServer::SQLiteIDBBackingStore::addExistingIndex):
(WebCore::IDBServer::SQLiteIDBBackingStore::createObjectStore):
(WebCore::IDBServer::SQLiteIDBBackingStore::createIndex):
(WebCore::IDBServer::SQLiteIDBBackingStore::uncheckedHasIndexRecord):
(WebCore::IDBServer::SQLiteIDBBackingStore::uncheckedPutIndexRecord):
(WebCore::IDBServer::SQLiteIDBBackingStore::keyExistsInObjectStore):
(WebCore::IDBServer::SQLiteIDBBackingStore::deleteRecord):
(WebCore::IDBServer::SQLiteIDBBackingStore::addRecord):
(WebCore::IDBServer::SQLiteIDBBackingStore::getRecord):
(WebCore::IDBServer::SQLiteIDBBackingStore::getAllObjectStoreRecords):
(WebCore::IDBServer::SQLiteIDBBackingStore::uncheckedGetIndexRecordForOneKey):
(WebCore::IDBServer::SQLiteIDBBackingStore::getCount):
* Source/WebCore/Modules/indexeddb/server/SQLiteIDBCursor.cpp:
(WebCore::IDBServer::SQLiteIDBCursor::bindArguments):
(WebCore::IDBServer::SQLiteIDBCursor::resetAndRebindPreIndexStatementIfNecessary):
* Source/WebCore/Modules/notifications/NotificationDataCocoa.mm:
(WebCore::NotificationData::fromDictionary):
* Source/WebCore/Modules/push-api/PushDatabase.cpp:
(WebCore::uuidToSpan):
* Source/WebCore/Modules/push-api/PushEvent.cpp:
(WebCore::dataFromPushMessageDataInit):
* Source/WebCore/Modules/push-api/PushManager.cpp:
(WebCore::PushManager::subscribe):
* Source/WebCore/Modules/webauthn/AuthenticationExtensionsClientOutputs.cpp:
(WebCore::AuthenticationExtensionsClientOutputs::toCBOR const):
* Source/WebCore/Modules/webauthn/AuthenticatorAttestationResponse.cpp:
(WebCore::coseKeyForAttestationObject):
(WebCore::AuthenticatorAttestationResponse::getAuthenticatorData const):
* Source/WebCore/Modules/webauthn/cbor/CBORValue.cpp:
(cbor::CBORValue::CBORValue):
* Source/WebCore/Modules/webauthn/cbor/CBORWriter.cpp:
(cbor::CBORWriter::encodeCBOR):
* Source/WebCore/Modules/webauthn/fido/Pin.cpp:
(fido::pin::SetPinRequest::tryCreate):
* Source/WebCore/Modules/webauthn/fido/U2fCommandConstructor.cpp:
(fido::WebCore::constructU2fSignCommand):
* Source/WebCore/Modules/webauthn/fido/U2fResponseConverter.cpp:
(fido::readU2fSignResponse):
* Source/WebCore/Modules/webcodecs/WebCodecsAudioData.cpp:
(WebCore::WebCodecsAudioData::create):
* Source/WebCore/Modules/webcodecs/WebCodecsEncodedAudioChunk.cpp:
(WebCore::WebCodecsEncodedAudioChunk::WebCodecsEncodedAudioChunk):
* Source/WebCore/Modules/webcodecs/WebCodecsEncodedVideoChunk.cpp:
(WebCore::WebCodecsEncodedVideoChunk::WebCodecsEncodedVideoChunk):
* Source/WebCore/Modules/websockets/WebSocket.cpp:
(WebCore::WebSocket::didReceiveMessage):
* Source/WebCore/Modules/websockets/WebSocketDeflateFramer.cpp:
(WebCore::WebSocketDeflateFramer::deflate):
(WebCore::WebSocketDeflateFramer::inflate):
* Source/WebCore/Modules/websockets/WebSocketDeflater.h:
(WebCore::WebSocketDeflater::span const):
(WebCore::WebSocketInflater::span const):
(WebCore::WebSocketDeflater::bytes const): Deleted.
(WebCore::WebSocketInflater::bytes const): Deleted.
* Source/WebCore/accessibility/mac/AccessibilityObjectMac.mm:
(WebCore::AXRemoteFrame::generateRemoteToken const):
* Source/WebCore/bindings/js/BufferSource.h:
(WebCore::BufferSource::span const):
(WebCore::BufferSource::bytes const): Deleted.
* Source/WebCore/bindings/js/JSDOMGlobalObject.cpp:
(WebCore::handleResponseOnStreamingAction):
* Source/WebCore/bindings/js/ScriptBufferSourceProvider.h:
* Source/WebCore/bindings/js/SerializedScriptValue.cpp:
(WebCore::CloneSerializer::dumpImageBitmap):
(WebCore::CloneSerializer::dumpIfTerminal):
(WebCore::CloneSerializer::write):
* Source/WebCore/contentextensions/ContentExtensionActions.cpp:
(WebCore::ContentExtensions::append):
* Source/WebCore/contentextensions/ContentExtensionStringSerialization.cpp:
(WebCore::ContentExtensions::serializeString):
* Source/WebCore/crypto/SubtleCrypto.cpp:
(WebCore::toKeyData):
(WebCore::copyToVector):
(WebCore::SubtleCrypto::wrapKey):
* Source/WebCore/crypto/cocoa/SerializedCryptoKeyWrapMac.mm:
(WebCore::wrapSerializedCryptoKey):
(WebCore::unwrapSerializedCryptoKey):
* Source/WebCore/crypto/parameters/CryptoAlgorithmAesCbcCfbParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmAesCtrParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmAesGcmParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmHkdfParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmPbkdf2Params.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmRsaKeyGenParams.h:
(WebCore::CryptoAlgorithmRsaKeyGenParams::publicExponentVector const):
* Source/WebCore/crypto/parameters/CryptoAlgorithmRsaOaepParams.h:
* Source/WebCore/dom/TextDecoder.cpp:
(WebCore::TextDecoder::decode):
* Source/WebCore/editing/cocoa/WebContentReaderCocoa.mm:
(WebCore::sanitizeMarkupWithArchive):
* Source/WebCore/fileapi/BlobBuilder.cpp:
(WebCore::BlobBuilder::append):
* Source/WebCore/inspector/agents/InspectorPageAgent.cpp:
(WebCore::InspectorPageAgent::mainResourceContent):
(WebCore::InspectorPageAgent::sharedBufferContent):
* Source/WebCore/loader/appcache/ApplicationCacheStorage.cpp:
(WebCore::ApplicationCacheStorage::store):
* Source/WebCore/loader/archive/mhtml/MHTMLParser.cpp:
(WebCore::MHTMLParser::parseNextPart):
* Source/WebCore/loader/cache/CachedFont.cpp:
(WebCore::CachedFont::policyForCustomFont):
* Source/WebCore/loader/cache/CachedScript.cpp:
(WebCore::CachedScript::script):
* Source/WebCore/platform/SharedBuffer.cpp:
(WebCore::combineSegmentsData):
(WebCore::FragmentedSharedBuffer::getContiguousData const):
(WebCore::FragmentedSharedBuffer::read const):
* Source/WebCore/platform/SharedBuffer.h:
(WebCore::DataSegment::span const):
(WebCore::SharedBuffer::span const):
(WebCore::DataSegment::bytes const): Deleted.
(WebCore::SharedBuffer::bytes const): Deleted.
* Source/WebCore/platform/cocoa/ContentFilterUnblockHandlerCocoa.mm:
(WebCore::ContentFilterUnblockHandler::webFilterEvaluatorData const):
* Source/WebCore/platform/graphics/ImageBackingStore.h:
(WebCore::ImageBackingStore::ImageBackingStore):
* Source/WebCore/platform/graphics/WOFFFileFormat.cpp:
(WebCore::convertWOFFToSfnt):
* Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateAVFoundationObjC.mm:
(WebCore::MediaPlayerPrivateAVFoundationObjC::shouldWaitForLoadingOfResource):
* Source/WebCore/platform/graphics/cocoa/WebMAudioUtilitiesCocoa.mm:
(WebCore::cookieFromOpusCookieContents):
* Source/WebCore/platform/graphics/coretext/FontCustomPlatformDataCoreText.cpp:
(WebCore::FontCustomPlatformData::serializedData const):
* Source/WebCore/platform/graphics/coretext/FontPlatformDataCoreText.cpp:
(WebCore::FontPlatformData::platformSerializationData const):
* Source/WebCore/platform/mediastream/mac/AVVideoCaptureSource.mm:
(WebCore::AVVideoCaptureSource::captureOutputDidFinishProcessingPhoto):
* Source/WebCore/platform/network/FormData.cpp:
(WebCore::FormData::create):
(WebCore::appendBlobResolved):
* Source/WebCore/platform/network/FormDataBuilder.cpp:
(WebCore::FormDataBuilder::append):
* Source/WebCore/platform/network/SynchronousLoaderClient.cpp:
(WebCore::SynchronousLoaderClient::didReceiveData):
* Source/WebCore/platform/network/cocoa/ResourceRequestCocoa.mm:
(WebCore::ResourceRequest::doUpdateResourceHTTPBody):
* Source/WebCore/platform/win/ClipboardUtilitiesWin.cpp:
(WebCore::append):
* Source/WebCore/testing/Internals.cpp:
(WebCore::Internals::deserializeBuffer const):
(WebCore::Internals::createPushSubscription):
* Source/WebCore/testing/MockContentFilter.cpp:
(WebCore::MockContentFilter::maybeDetermineStatus):
* Source/WebCore/testing/ServiceWorkerInternals.cpp:
(WebCore::ServiceWorkerInternals::schedulePushEvent):
(WebCore::ServiceWorkerInternals::createPushSubscription):
* Source/WebCore/workers/service/background-fetch/BackgroundFetch.cpp:
(WebCore::BackgroundFetch::doStore):
* Source/WebCore/xml/XMLHttpRequest.cpp:
(WebCore::XMLHttpRequest::send):
* Source/WebCore/xml/parser/XMLDocumentParserLibxml2.cpp:
(WebCore::openFunc):
* Source/WebGPU/WGSL/ConstantFunctions.h:
(WGSL::constantMatrix):
(WGSL::CONSTANT_FUNCTION):
* Source/WebGPU/WGSL/ConstantValue.h:
(WGSL::ConstantValue::vector const):
(WGSL::ConstantValue::toVector const): Deleted.
* Source/WebGPU/WGSL/Types.cpp:
(WGSL::conversionRank):
* Source/WebKit/NetworkProcess/Downloads/cocoa/DownloadCocoa.mm:
(WebKit::Download::platformCancelNetworkLoad):
* Source/WebKit/NetworkProcess/NetworkResourceLoader.cpp:
(WebKit::sendReplyToSynchronousRequest):
* Source/WebKit/NetworkProcess/cocoa/NetworkSessionCocoa.mm:
(-[WKNetworkSessionDelegate URLSession:task:didCompleteWithError:]):
(WebKit::NetworkSessionCocoa::setProxyConfigData):
* Source/WebKit/NetworkProcess/cocoa/WebSocketTaskCocoa.mm:
(WebKit::WebSocketTask::readNextMessage):
* Source/WebKit/NetworkProcess/curl/WebSocketTaskCurl.cpp:
(WebKit::WebSocketTask::appendReceivedBuffer):
(WebKit::WebSocketTask::sendClosingHandshakeIfNeeded):
* Source/WebKit/NetworkProcess/storage/CacheStorageDiskStore.cpp:
(WebKit::CacheStorageDiskStore::readAllRecordInfos):
(WebKit::CacheStorageDiskStore::readRecords):
(WebKit::encodeRecordHeader):
(WebKit::encodeRecordBody):
(WebKit::encodeRecord):
* Source/WebKit/NetworkProcess/storage/CacheStorageManager.cpp:
(WebKit::writeSizeFile):
* Source/WebKit/Shared/API/APIData.h:
(API::Data::span const):
(API::Data::bytes const): Deleted.
* Source/WebKit/Shared/API/APIData.serialization.in:
* Source/WebKit/Shared/API/Cocoa/WKRemoteObjectCoder.mm:
(-[WKRemoteObjectDecoder decodeBytesForKey:returnedLength:]):
* Source/WebKit/Shared/API/c/WKData.cpp:
(WKDataGetBytes):
* Source/WebKit/Shared/APIWebArchive.mm:
(API::WebArchive::WebArchive):
* Source/WebKit/Shared/APIWebArchiveResource.mm:
(API::WebArchiveResource::WebArchiveResource):
* Source/WebKit/Shared/Cocoa/CoreIPCCFURL.h:
* Source/WebKit/Shared/Cocoa/CoreIPCCFURL.mm:
(WebKit::CoreIPCCFURL::vector const):
(WebKit::CoreIPCCFURL::bytes const): Deleted.
* Source/WebKit/Shared/Cocoa/CoreIPCCFURL.serialization.in:
* Source/WebKit/Shared/Cocoa/SandboxExtensionCocoa.mm:
(WebKit::SandboxExtension::createHandleForTemporaryFile):
* Source/WebKit/Shared/Cocoa/WKNSData.mm:
(-[WKNSData bytes]):
* Source/WebKit/Shared/Cocoa/WebPushMessageCocoa.mm:
(WebKit::WebPushMessage::fromDictionary):
* Source/WebKit/Shared/WTFArgumentCoders.serialization.in:
* Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in:
* Source/WebKit/Shared/WebPushMessage.cpp:
(WebKit::WebPushMessage::notificationPayloadToCoreData const):
* Source/WebKit/Shared/cf/CookieStorageUtilsCF.mm:
(WebKit::identifyingDataFromCookieStorage):
* Source/WebKit/Shared/mac/AuxiliaryProcessMac.mm:
(WebKit::compileAndCacheSandboxProfile):
* Source/WebKit/UIProcess/API/C/WKNotification.cpp:
(WKNotificationCopyCoreIDForTesting):
* Source/WebKit/UIProcess/API/C/WKNotificationManager.cpp:
(WKNotificationManagerProviderDidClickNotification_b):
* Source/WebKit/UIProcess/API/C/WKPage.cpp:
(WKPageLoadData):
(WKPageLoadDataWithUserData):
(restoreFromSessionState):
* Source/WebKit/UIProcess/API/C/WKSessionStateRef.cpp:
(WKSessionStateCreateFromData):
* Source/WebKit/UIProcess/API/Cocoa/WKProcessPool.mm:
(-[WKProcessPool _setObject:forBundleParameter:]):
(-[WKProcessPool _setObjectsForBundleParametersWithDictionary:]):
* Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm:
(-[WKWebView _restoreFromSessionStateData:]):
* Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataStore.mm:
(-[WKWebsiteDataStore setProxyConfigurations:]):
* Source/WebKit/UIProcess/API/Cocoa/_WKWebAuthenticationPanel.mm:
(getAllLocalAuthenticatorCredentialsImpl):
(+[_WKWebAuthenticationPanel setDisplayNameForLocalCredentialWithGroupAndID:credential:displayName:]):
(+[_WKWebAuthenticationPanel setNameForLocalCredentialWithGroupAndID:credential:name:]):
(+[_WKWebAuthenticationPanel importLocalAuthenticatorWithAccessGroup:credential:error:]):
(+[_WKWebAuthenticationPanel exportLocalAuthenticatorCredentialWithGroupAndID:credential:error:]):
(-[_WKWebAuthenticationPanel makeCredentialWithMediationRequirement:clientDataHash:options:completionHandler:]):
(-[_WKWebAuthenticationPanel getAssertionWithMediationRequirement:clientDataHash:options:completionHandler:]):
(+[_WKWebAuthenticationPanel encodeMakeCredentialCommandWithClientDataHash:options:userVerificationAvailability:authenticatorSupportedExtensions:]):
(+[_WKWebAuthenticationPanel encodeGetAssertionCommandWithClientDataHash:options:userVerificationAvailability:authenticatorSupportedExtensions:]):
* Source/WebKit/UIProcess/API/glib/WebKitFaviconDatabase.cpp:
(webkitFaviconDatabaseSetIconForPageURL):
* Source/WebKit/UIProcess/API/glib/WebKitWebResource.cpp:
(resourceDataCallback):
(webkit_web_resource_get_data_finish):
* Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp:
(getContentsAsMHTMLDataCallback):
(webkit_web_view_save_finish):
* Source/WebKit/UIProcess/Cocoa/LegacyCustomProtocolManagerClient.mm:
(-[WKCustomProtocolLoader connection:didReceiveData:]):
* Source/WebKit/UIProcess/Cocoa/SOAuthorization/SubFrameSOAuthorizationSession.mm:
(WebKit::SubFrameSOAuthorizationSession::completeInternal):
* Source/WebKit/UIProcess/Notifications/WebNotificationManagerProxy.cpp:
(WebKit::WebNotificationManagerProxy::providerDidCloseNotifications):
* Source/WebKit/UIProcess/WebAuthentication/Cocoa/CcidConnection.mm:
(WebKit::CcidConnection::transact const):
* Source/WebKit/UIProcess/WebAuthentication/Cocoa/LocalAuthenticator.mm:
(WebKit::LocalAuthenticatorInternal::getExistingCredentials):
(WebKit::LocalAuthenticator::processLargeBlobExtension):
(WebKit::LocalAuthenticator::continueMakeCredentialAfterAttested):
* Source/WebKit/UIProcess/WebAuthentication/Cocoa/NfcConnection.mm:
(WebKit::NfcConnection::transact const):
* Source/WebKit/UIProcess/WebAuthentication/Cocoa/WebAuthenticatorCoordinatorProxy.mm:
(WebKit::toExtensionOutputs):
* Source/WebKit/UIProcess/WebAuthentication/Virtual/VirtualAuthenticatorUtils.mm:
(WebKit::signatureForPrivateKey):
* Source/WebKit/UIProcess/WebPageProxy.cpp:
(WebKit::WebPageProxy::didChooseFilesForOpenPanelWithDisplayStringAndIcon):
(WebKit::WebPageProxy::getWebCryptoMasterKey):
* Source/WebKit/UIProcess/WebURLSchemeTask.cpp:
(WebKit::WebURLSchemeTask::didComplete):
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp:
(WebKit::WebsiteDataStore::resumeDownload):
* Source/WebKit/UIProcess/ios/WKContentView.mm:
(-[WKContentView _accessibilityRegisterUIProcessTokens]):
* Source/WebKit/UIProcess/mac/WKPrintingView.mm:
(-[WKPrintingView _preparePDFDataForPrintingOnSecondaryThread]):
* Source/WebKit/UIProcess/mac/WKSharingServicePickerDelegate.mm:
(-[WKSharingServicePickerDelegate sharingService:didShareItems:]):
* Source/WebKit/UIProcess/mac/WebContextMenuProxyMac.mm:
(WebKit::WebContextMenuProxyMac::removeBackgroundFromControlledImage):
* Source/WebKit/UIProcess/mac/WebViewImpl.mm:
(WebKit::WebViewImpl::accessibilityRegisterUIProcessTokens):
* Source/WebKit/WebProcess/InjectedBundle/API/c/WKBundle.cpp:
(WKBundleCopyWebNotificationID):
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageEditorClient.cpp:
(WebKit::InjectedBundlePageEditorClient::getPasteboardDataForRange):
* Source/WebKit/WebProcess/InjectedBundle/mac/InjectedBundleMac.mm:
(WebKit::createUnarchiver):
* Source/WebKit/WebProcess/Network/WebSocketChannel.cpp:
(WebKit::WebSocketChannel::createMessageQueue):
* Source/WebKit/WebProcess/Network/webrtc/RTCDataChannelRemoteManager.cpp:
(WebKit::RTCDataChannelRemoteManager::RemoteSourceConnection::didReceiveStringData):
* Source/WebKit/WebProcess/Plugins/PDF/PDFIncrementalLoader.mm:
(WebKit::PDFPluginStreamLoaderClient::didReceiveData):
* Source/WebKit/WebProcess/WebPage/Cocoa/WebPageCocoa.mm:
(WebKit::WebPage::accessibilityTransferRemoteToken):
(WebKit::WebPage::bindRemoteAccessibilityFrames):
* Source/WebKit/WebProcess/WebPage/WebFrame.cpp:
(WebKit::WebFrame::source const):
* Source/WebKit/WebProcess/WebPage/mac/WebPageMac.mm:
(WebKit::WebPage::handleSelectionServiceClick):
* Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm:
(WebKit::registerLogHook):
* Source/WebKit/webpushd/ApplePushServiceConnection.mm:
(-[_WKAPSConnectionDelegate connection:didReceivePublicToken:]):
* Source/WebKit/webpushd/WebPushDaemon.mm:
(WebPushD::WebPushDaemon::getPendingPushMessages):
(WebPushD::WebPushDaemon::setPublicTokenForTesting):
* Source/WebKitLegacy/WebCoreSupport/SocketStreamHandle.cpp:
(WebCore::SocketStreamHandle::sendHandshake):
* Source/WebKitLegacy/WebCoreSupport/SocketStreamHandleImpl.cpp:
(WebCore::cookieDataForHandshake):
* Source/WebKitLegacy/WebCoreSupport/WebSocketChannel.cpp:
(WebCore::WebSocketChannel::send):
(WebCore::WebSocketChannel::startClosingHandshake):
(WebCore::WebSocketChannel::processOutgoingFrameQueue):
* Source/WebKitLegacy/mac/WebView/WebFrame.mm:
(+[WebFrame stringWithData:textEncodingName:]):
* Source/WebKitLegacy/mac/WebView/WebHTMLRepresentation.mm:
(-[WebHTMLRepresentation documentSource]):
* Source/WebKitLegacy/mac/WebView/WebResource.mm:
(-[WebResource _stringValue]):
* Tools/TestWebKitAPI/Tests/WTF/cf/VectorCF.cpp:
(TestWebKitAPI::TEST):
* Tools/TestWebKitAPI/Tests/WebCore/CtapResponseTest.cpp:
(TestWebKitAPI::TEST):
* Tools/TestWebKitAPI/Tests/WebCore/KeyedCoding.cpp:
(TestWebKitAPI::TEST):
(TestWebKitAPI::testSimpleValue):
* Tools/TestWebKitAPI/Tests/WebCore/curl/CurlMultipartHandleTests.cpp:
(TestWebKitAPI::Curl::span):
(TestWebKitAPI::Curl::TEST):
(TestWebKitAPI::Curl::toSpan): Deleted.
* Tools/TestWebKitAPI/Tests/WebKitCocoa/EventAttribution.mm:
(TestWebKitAPI::signUnlinkableTokenAndSendSecretToken):
* Tools/TestWebKitAPI/Tests/WebKitCocoa/MediaLoading.mm:
(TestWebKitAPI::testVideoBytes):
(TestWebKitAPI::TEST):
* Tools/TestWebKitAPI/Tests/WebKitCocoa/WebsiteDataStoreCustomPaths.mm:
(respondToRangeRequests):
* Tools/TestWebKitAPI/cocoa/HTTPServer.mm:
(TestWebKitAPI::appendUTF8ToVector):
* Tools/WebKitTestRunner/DataFunctions.h:
(WTR::uuidToData):

Canonical link: https://commits.webkit.org/276520@main
